Scripts/book.js

```diff
@@ -1,15 +1,13 @@
 /* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1: